public/manifest.json

```diff
@@ -1,15 +1,45 @@
 {
-  "short_name": "React App",
-  "name": "Create React App Sample",
+  "short_name": "Core Units Expenses",
+  "name": "MakerDAO Expenses",
   "icons": [
     {
-      "src": "favicon.svg",
-      "sizes": "64x64 32x32 24x24 16x16",
-      "type": "image/svg"
+      "src": "/icons/Android-MakerDAO-Expenses-bars-color-square-192x192.png",
+      "type": "image/png",
+      "sizes": "192x192"
```

Comment:
Maybe could be nice to compress a bit the images, the smallest has ~50k but the medium sizes have +400kb. https://tinypng.com/ does an excellent job, it reduces about 70% the size without compromising quality

Reply:
ok,  i will reduce the weight of the image